### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by MarcosCarballal** [kdickerson/automatonSimulator#1]
Hi there! It would be awesome if you could add some kind of comments to be able to drag around and explain the logic of the DFA. Thanks!

**Comment by kdickerson** [comment]
You're right, that would be useful. Next time I get time to work on this I
will probably add it in some form.

**Comment by MarcosCarballal** [comment]
Also please allow more than 5 connections! It really helps thank you so much

**Comment by kdickerson** [comment]
Increased maxConnections to 10.

I'm looking in to adding comments; it doesn't fit in the current design very easily, but I'll keep thinking about it.

**Comment by AlexHelmutSonntag** [comment]
Hi! Great job on the Simulator. I am not sure if it's an issue with my device but the whole app freezes when I try to use the manual tester. It goes into a non-responding state. I tried 2 different browsers and used a new incognito window to make sure it has nothing to do with the cache. It would be really nice if you could direct me to a solution. Thanks!
